modules/dxp/apps/scim/scim-configuration-web/src/main/java/com/liferay/scim/configuration/web/internal/portlet/action/SaveScimConfigurationMVCActionCommand.java

```diff
@@ -155,6 +169,48 @@ else if (Objects.equals(cmd, "reset")) {
 
 			_oAuth2ApplicationLocalService.deleteOAuth2Application(
 				oAuth2Application);
+
+			Indexer<User> userIndexer = IndexerRegistryUtil.nullSafeGetIndexer(
+				User.class);
+
+			Indexer<UserGroup> userGroupIndexer =
+				IndexerRegistryUtil.nullSafeGetIndexer(UserGroup.class);
+
+			ActionableDynamicQuery query =
+				_expandoValueLocalService.getActionableDynamicQuery();
+
+			query.setAddCriteriaMethod(
+				dynamicQuery -> {
+					Property dataProperty = PropertyFactoryUtil.forName("data");
```

Comment:
Shouldn't this be "data_"?  See https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portlet/expando/model/impl/ExpandoValueModelImpl.java#L81

Reply:
As you mentioned, "data" works, but "data_" doesn't, even though it is the true column name.

I see other examples where the column ends with an underscore, but it is not included in the `dataProperty`, so your approach is correct.  (See "type_" for the `group_` table)